src/HotChocolate/Fusion-vnext/src/Fusion.Composition/Properties/CompositionResources.resx

```diff
@@ -60,6 +60,9 @@
   <data name="LogEntryHelper_KeyInvalidSyntax" xml:space="preserve">
     <value>A @key directive on type '{0}' in schema '{1}' contains invalid syntax in the 'fields' argument.</value>
   </data>
+  <data name="LogEntryHelper_ProvidesOnNonCompositeField" xml:space="preserve">
+    <value>A @provides directive on field '{0}' in schema '{1}' is specified on a non-composite field, which cannot be provided.</value>
```

Comment:
Maybe something like this?

`The field 'User.email' in schema 'A' includes a @provides directive, but does not return a composite type.`